lib/doggy/cli.rb

```diff
@@ -6,6 +6,13 @@ module Doggy
   class CLI < Thor
     include Thor::Actions
 
+    map %w[--version -v] => :__print_version
+
+    desc '--version, -v', 'print the version'
```

Comment:
yeah legacy thing from the previous contributor, I'm moving them to default to `'` slowly.